Samples/Package/Out-of-Box/Calculator-OOB-Sample/scripts/outofbox/launch.ps1

```diff
@@ -14,30 +14,39 @@ $log_dir = "$root_dir\logs"
 
 $log_file = "$log_dir\$script_name.log"
 
-if(-not (test-path -path $log_dir )) {
-    New-Item -itemtype directory -path $log_dir
+if (-not (test-path -path $log_dir )) {
+   New-Item -itemtype directory -path $log_dir
 }
 
 Function Log {
    Param ([string]$log_string)
    Write-Host $log_string
    Add-Content $log_file -value $log_string
 }
+$appName = ''
+$appProcessName = ''
+Get-ChildItem -Filter "*.json" | ForEach-Object {
+   $jsonInfo = (Get-Content $_.fullname | ConvertFrom-Json)
+   $appName = $jsonInfo.appName;  
+   $appProcessName = $jsonInfo.appProcessName;  
+}
+Write-Host "AppName:$appName"
+Write-Host "AppProcessName:$appProcessName"
 
 # Step 1: Launch the application
 # For example:
 #    - If your application is under bin folder: Start-Process -FilePath "$bin_dir\MyApp.exe"
 #    - Use environment variable if need: Start-Process -FilePath "$env:comspec" -ArgumentList "/c dir `"%systemdrive%\program files`""
 Log("Launch Application")
 # Change the $exePath to your execution path, add -ArgumentList if need.
-$exePath = "C:\Program Files (x86)\Calculator\calculator.exe"
+$exePath = "C:\Program Files (x86)\Calculator\$appName"
```

Comment:
"Calculator" should not be in a common script.
Put the full path to the config file.